docs/FAQ.md

```diff
@@ -4,6 +4,10 @@ title: OpenEBS FAQ
 sidebar_label: FAQ
 ---
 
+<center><p style="padding: 20px; margin: 20px 0; border-radius: 3px; background-color: #eeeeee;"><strong>
+  Documentation for OpenEBS v0.5 is no longer actively maintained. The version you are currently viewing is a static snapshot. Click here for [latest](https://docs.openebs.io) version.
```

Comment:
Click here for the [latest](...) version.

Reply:
Addressed. 
Thank You.